src/ensembl/src/shared/components/panel/Panel.test.tsx

```diff
@@ -40,71 +41,68 @@ const defaultProps: PanelProps = {
-  return <Panel {...defaultProps} {...props} />;
+  return render(<Panel {...defaultProps} {...props} />);
 };
 
 describe('<Tabs />', () => {
-  let wrapper: any;
-
-  afterEach(() => {
+  beforeEach(() => {
     jest.resetAllMocks();
   });
 
-  it('displays the HTML header', () => {
-    wrapper = mount(renderPanel());
+  it('displays the header with the correct content', () => {
+    const { container } = renderPanel();
+    const header = container.querySelector('.header');
 
-    expect(
-      wrapper.find('.header').children().contains(defaultProps.header)
-    ).toBeTruthy();
+    expect(header?.textContent).toBe(defaultProps.header);
   });
 
-  it('displays the string header', () => {
-    const stringHeader = faker.lorem.words();
-
-    wrapper = mount(renderPanel({ header: stringHeader }));
-    expect(wrapper.find('.header').text()).toBe(stringHeader);
-  });
+  it('displays the body with the correct content', () => {
+    const { container } = renderPanel();
+    const body = container.querySelector('.body');
 
-  it('displays the body content', () => {
-    wrapper = mount(renderPanel());
-    expect(wrapper.find('.body').contains(defaultProps.children)).toBeTruthy();
+    expect(body?.textContent).toBe(defaultProps.children);
   });
 
   it('displays the close button only if onClose is set', () => {
-    wrapper = mount(renderPanel({ onClose }));
-    expect(wrapper.find('.closeButton')).toHaveLength(1);
+    const { container } = renderPanel({ onClose });
+    const closeButton = container.querySelector('.closeButton');
+
+    expect(closeButton).toBeTruthy();
   });
 
   it('does not display the close button if onClose is not set', () => {
-    wrapper = mount(renderPanel());
-    expect(wrapper.find('.closeButton')).toHaveLength(0);
+    const { container } = renderPanel();
+    const closeButton = container.querySelector('.closeButton');
+
+    expect(closeButton).toBeFalsy();
   });
 
   it('calls the onClose function when the close button is clicked', () => {
-    wrapper = mount(renderPanel({ onClose }));
-    wrapper.find('.closeButton').simulate('click');
+    const { container } = renderPanel({ onClose });
+    const closeButton = container.querySelector('.closeButton');
+
+    userEvent.click(closeButton as HTMLElement);
 
     expect(onClose).toHaveBeenCalled();
   });
 
   it('applies the passed in classes', () => {
-    wrapper = mount(renderPanel());
-
-    expect(
-      wrapper.find('.panel').first().hasClass(panelClassNames.panel)
-    ).toBeTruthy();
-    expect(
-      wrapper.find('.body').first().hasClass(panelClassNames.body)
-    ).toBeTruthy();
-    expect(
-      wrapper.find('.header').first().hasClass(panelClassNames.header)
-    ).toBeTruthy();
+    const { container } = renderPanel();
+    const panel = container.querySelector('.panel') as HTMLElement;
+    const panelBody = container.querySelector('.body') as HTMLElement;
+    const panelHeader = container.querySelector('.header') as HTMLElement;
```

Comment:
Since `panelHeader` is already defined on line 26, should we use a different variable name here?

Reply:
🤷‍♂️ 

I've updated the names of variables on lines 24-26 if that helps